testing-tools/Dockerfile

```diff
@@ -25,49 +28,54 @@ SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
 # This is needed so that krb5-user installs without prompting for a realm.
 ENV DEBIAN_FRONTEND=noninteractive
 
-# krb5-user/libkrb5-dev are needed for Kerberos support.
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-        build-essential \
-        ca-certificates \
-        curl \
-        gzip \
-        jq \
-        krb5-user \
-        kubernetes-client \
-        libkrb5-dev \
-        libssl-dev \
-        libxml2-dev \
-        libxslt1-dev \
-        pkg-config \
-        python3-certifi \
-        python3-idna \
-        python3-semver \
-        python3-thrift \
-        python3-toml \
-        python3-urllib3 \
-        tar \
-        zip \
-        unzip \
-        # Java 11 seems like the best middle-ground for all tools
```

Comment:
nit: I still dislike us commenting like 20 lines above and prefer keeping the comments at the line where they apply.
Not blocking this PR

Reply:
That's not possible with the EOF/heredoc style unfortunately.

Reply:
I did not know that syntax, thanks.,

[`d5ef172` (#1192)](https://github.com/stackabletech/docker-images/pull/1192/commits/d5ef1725100fb682f27a9a18565167608a69070d)